.github/workflows/ci.yml

```diff
@@ -32,19 +32,26 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5
+        uses: actions/setup-python@9e62be81b28222addecf85e47571213eb7680449
```

Comment:
This should no longer be needed, since the latest `setup-python` release now includes support. Revert this change?